Automattic/wp-calypso
# Fix/purchase price format

The display text price can be formatted incorrectly because;

* When Calypso creates the refund amount price, the result is a float without any specified precision
* The price text used in manage purchase page is passed from Store_Subscription and does not format decimal prices correctly

<img width="704" alt="example badly formatted prices" src="https://user-images.githubusercontent.com/5560595/49450910-ed87e100-f7d5-11e8-8115-dfdaa317e153.png">

This patch works with D21774-code to correct the display prices.

#### Testing instructions

* Log in with SSP to username eoigalwpcomtest5
* Go to http://calypso.localhost:3000/me/purchases/eoigalwpcomtest5.wordpress.com/577567
* Verify the plan prices are formatted correctly
* Go to http://calypso.localhost:3000/me/purchases/eoigalwpcomtest5.wordpress.com/577567/cancel
* Verify the refund amounts are formatted correctly

Fixes #
* Refund amounts in the cancellation notice
* Plan details price
